lib/storage/storage.go

```diff
@@ -246,7 +243,7 @@ func MustOpenStorage(path string, retentionMsecs int64, maxHourlySeries, maxDail
 
 	// Initialize nextRotationTimestamp
 	nowSecs := time.Now().UnixNano() / 1e9
-	nextRotationTimestamp := nextRetentionDeadlineSeconds(nowSecs, retentionMsecs/1000, retentionTimezoneOffsetSecs)
+	nextRotationTimestamp := nextRetentionDeadlineSeconds(nowSecs, retention.Milliseconds()/1000, retentionTimezoneOffsetSecs)
```

Comment:
```suggestion
	nextRotationTimestamp := nextRetentionDeadlineSeconds(nowSecs, retention.Second(), retentionTimezoneOffsetSecs)
```

Reply:
No, the Seconds() returns float64, and does more things inside (although pretty fast probably, but still unnecessary conversion to float and back).

Reply:
Added comment to code